docs/howtos/DataPlane Generation - DPG/04renaming.mdx

```diff
@@ -212,7 +212,10 @@ using Azure.ClientGenerator.Core;
 <TabItem value="python" label="Python">
 
 ```python
-client.get_computed_model()
+class InputModel:
```

Comment:
```suggestion
class ReadModel:
```

Reply:
Get model is the method name, not the model name, why we rename the mode?

Reply:
oh, my fault.